### PR TITLE
[1.14] Fix missing image names on list

### PR DESCRIPTION
**Pull request by saschagrunert**
Backport of #2455

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
/approve

**Comment by haircommander** [comment]
LGTM

File: server/image_list.go [hunk 1]
```diff
@@ -31,22 +32,43 @@ func (s *Server) ListImages(ctx context.Context, req *pb.ListImagesRequest) (res
...
+
+	repoTags := []string{"<none>:<none>"}
+	if len(from.RepoTags) > 0 {
+		repoTags = from.RepoTags
+	}
+
+	repoDigests := []string{"<none>@<none>"}
+	if len(from.RepoDigests) > 0 {
+		repoDigests = from.RepoDigests
+	}
```
Comment (mtrmac) [inline server/image_list.go]: I can’t see how this makes sense: https://github.com/cri-o/cri-o/pull/2455#discussion_r291677114 .

**Pull request merged**